index.js

```diff
@@ -0,0 +1,30 @@
+function scoreCalculator(arr) {
+  let score = 0;
+  let bonus = arr.every(num => num < 5);
+
+  for (var i = 0; i < arr.length; i++) {
+
+    if (arr[i] >= 5 && arr[i] <= 10) {
+      score += 5;
+
+    } else if (arr[i] < 5) {
+      score += 10;
+
+      if (bonus) {
```

Comment:
This could potentially add the bonus multiple times if you get a lot of scores under 5. It's best to do the bonus just once outside the loop.